upstream/CVE-2019-11243.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "CVE-2019-11243",
+  "modified": "2019-04-18T21:31:53Z",
+  "published": "2019-04-18T21:31:53Z",
+  "summary": "rest.AnonymousClientConfig() does not remove the serviceaccount credentials from config created by rest.InClusterConfig()",
+  "details": "In Kubernetes v1.12.0-v1.12.4 and v1.13.0, the rest.AnonymousClientConfig() method returns a copy of the provided config, with credentials removed (bearer token, username/password, and client certificate/key data). In the affected versions, rest.AnonymousClientConfig() did not effectively clear service account credentials loaded using rest.InClusterConfig()",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubernetes"
```

Comment:
I carefully read the description in GitHub. I believe it affects `k8s.io/client-go` only.